airflow/cli/cli_config.py

```diff
@@ -561,6 +561,7 @@ def string_lower_type(val):
 # kerberos
 ARG_PRINCIPAL = Arg(("principal",), help="kerberos principal", nargs="?")
 ARG_KEYTAB = Arg(("-k", "--keytab"), help="keytab", nargs="?", default=conf.get("kerberos", "keytab"))
+ARG_KERBEROS_MODE = Arg(("-m", "--mode"), help="mode to run airflow kerberos", nargs="?", default="daemon")
```

Comment:
Looking at the screenshots. Since we also have `--daemon` mode using deamon here as well is misleding - so likely we should come with a different name. Mabye simply we should call it "standard" mode ?

Reply:
Yeah it could be confusing. What do you suggest? 
Standard vs one-time

Reply:
Or maybe forever vs one-time
long vs one-time?

Reply:
What if we didn't have "mode", but a "one-time" flag instead. I doubt we will have another mode down the road and sidesteps the "normal/daemon/standard" naming problem.

Reply:
This is nice. I am going to make this change!